src/main.kt

```diff
@@ -14,6 +13,10 @@ class DokkaArguments {
     ValueDescription("<path>")
     public var src: String = ""
 
+    Argument(value = "srcLink", description = "Mapping between a source directory and a Web site for browsing the code")
+    ValueDescription("<path>=<url>[#lineSuffix]")
+    public var srcLink: String = ""
+
```

Comment:
The underlying logic supports multiple mappings, but I'm not such a big fan of trying to represent this in a command line parameter.